web/beacon-app/src/features/projects/components/Settings.tsx

```diff
@@ -20,9 +20,11 @@ const ProjectSettings = () => {
 
   return (
     <>
-      <div className="flex w-full justify-end">
+      <div>
         <Button variant="ghost" className="flex-end bg-inherit w-8 border-none" onClick={open}>
-          <SettingIcon />
+          <div className="ml-[4.5rem]">
```

Comment:
This should make it so that the cogwheel isn't so far away from the end of the border and make it match the other side a little better.